pkg/api/customization/project/project_actions.go

```diff
@@ -136,8 +138,17 @@ func (h *Handler) viewMonitoring(actionName string, action *types.Action, apiCon
 		return httperror.NewAPIError(httperror.InvalidState, "disabling Monitoring")
 	}
 
+	appName, appTargetNamespace := monitoring.ProjectMonitoringInfo(project.Name)
+	appList, err := h.AppClient.List(monitoring.OwnedAppListOptions(project.Spec.ClusterName, appName, appTargetNamespace))
```

Comment:
Same as cluster action comment, I think we should use `Client.Get` instead.

Reply:
updated